asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
Are you counting only in comments or also in code?  In the latter case, of course "tailcall" is used in many identifiers, but that doesn't make it a correct usage of the English language.  More generally, frequence of use doesn't imply correctness.  Here, "tail call" is the correct form (as a noun), or "tail-call" as an adjective.